tools/anvil-join-anvil

```diff
@@ -1641,6 +1641,16 @@ sub wait_for_access
 							network => $this_network, 
 							peer_ip => $peer_ip_address,
 						}});
+						
+						my $job_created = check_for_bad_key($anvil, $peer_ip_address);
+						$anvil->Log->variables({source => $THIS_FILE, line => __LINE__, level => 2, list => { job_created => $job_created }});
+						
+						if ($job_created)
+						{
+							# Sleep for ten seconds to give the job time to get 
+							# picked up and run.
+							sleep 10;
```

Comment:
Is there a reason to not directly call the script?

The job is registered for the local machine (judging by `Get->host_uuid` when making the job) + sleep a guessed amount of time to remove the key(s).

Reply:
Not particularly, though if I did, it's not as easy to track the progress as `System->call` doesn't normally return until the called program exits. I might have had another reason? If so though, I can't recall atm